.coveragerc

```diff
@@ -1,3 +1,10 @@
 [run]
 branch = True
-source = lona
\ No newline at end of file
+source = lona
+
+omit =
+    lona/__main__.py
+    lona/default_routes.py
+    lona/logging.py
+    lona/command_line/*
+    lona/shell/*
```

Comment:
im not quite sure why these would absolutely need to be omitted from coverage. testing CLI-args and shel lcommands seems like something that we would want long-term.

Reply:
Hm, you are right. I removed everything but ``lona/default_routes.py`` and ``lona/__main__.py``